scss/_patterns_navigation.scss

```diff
@@ -100,31 +127,37 @@
       }
 
       .p-navigation__link {
```

Comment:
- Should we have a ```external``` class as we have this pattern on some websites already?
- When hovering on a ```.p-navigation__link``` the cursor stays at default until you are over the text.
 
![2017-05-03 11 14 40](https://cloud.githubusercontent.com/assets/1201618/25656619/cb767570-2ff1-11e7-80e3-f541e0ba590d.gif)